front/src/applications/editor/components/EntitySumUp.tsx

```diff
@@ -328,12 +328,7 @@ const EntitySumUp = ({ entity, id, objType, classes, status, error }: EntitySumU
       setState({ type: 'loading' });
 
       if (!entity) {
-        entity = await getEntity(
-          infraID as number,
-          id as string,
-          objType as EditoastType,
-          dispatch
-        );
+        entity = await getEntity(infraID as number, id, objType as EditoastType, dispatch);
```

Comment:
I think there's a type mismatch on `id`. `getEntity` expects `id` as a `string`, but id can be `undefined`.

Reply:
Ah, it's my fault. My editor complained, and it misled me. But you're right.